### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: grails-app/controllers/wekb/PublicController.groovy [hunk 1]
```diff
@@ -220,26 +220,27 @@ class PublicController {
...
-            rowMap.field = it
+            rowMap.field = it.value
             rowData << rowMap
           }
           columnData << rowData
         }
-
-        XSSFWorkbook workbook = exportService.generateXLSXWorkbook('Package_Export', export.titleRow, columnData)
+        */
+        XSSFWorkbook workbook = exportService.generateXLSXWorkbook('Package_Export', export.titleRow.toList(), export.columnData.toList())
```
Comment (croqueGrec09) [inline grails-app/controllers/wekb/PublicController.groovy:243]: toList() in a Map<String, List> ?

File: grails-app/services/wekb/ExportService.groovy [hunk 2]
```diff
@@ -519,6 +528,45 @@ class ExportService {
 
     }
 
+    Map<String,List> exportPackageTippsAsTSVWithSQL(Package pkg, List status) {
+        log.debug("Begin exportPackageTippsAsTSVWithSQL")
+        DataSource dataSource = BeanStore.getDataSource()
+        Sql sql = new Sql(dataSource)
+        Map<String, String> titleHeaders = getTitleHeadersWithSQL()
+        Map<String, List> export = [titleRow: titleHeaders.keySet()]
```
Comment (croqueGrec09) [inline grails-app/services/wekb/ExportService.groovy:536]: would it not be more convenient to use Map<String, Collection> instead?

**Review by croqueGrec09 — COMMENTED** [review]
beauty remarks

File: grails-app/services/wekb/ExportService.groovy (hunk 2 above)
Comment (Moetez16) [reply grails-app/services/wekb/ExportService.groovy:536]: Which ? The First or second?

**Pull request merged**